README.md

```diff
@@ -225,6 +225,22 @@ Must be set along with `--read_from_info_schema`. <br>
 Defaults to 1.
 </ul>
 
+`--read_from_info_schema_start_time="start-timestamp"` <br>
+`--read_from_info_schema_end_time="end-timestamp"`
+<ul>
+Alternative to `read_from_info_schema_days` option,<br>
+to specify start and end date or timestamp of INFORMATION_SCHEMA to read.<br>
+Must be set along with `--read_from_info_schema`. <br>
+Defaults to `--read_from_info_schema_days` option.
+</ul>
+
+`--read_from_info_schema_timeout_in_secs=n`
+<ul>
+Specifies timeout, in secs, to query INFORMATION SCHEMA<br>
+Must be set along with `--read_from_info_schema`. <br>
+Defaults to 10.
```

Comment:
Could we default it to something higher, maybe 60?

Reply:
The rationale for 10 seconds is based on the default timeout for the QueryJob (Ref: 
https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/query#:~:text=By%20default%2C%20this%20limit%20is,period%20in%20the%20timeoutMs%20field.
).

sure, will change it to 60 seconds.